src/app/src/main/kotlin/org/vaccineimpact/api/app/repositories/jooq/JooqTouchstoneRepository.kt

```diff
@@ -208,4 +329,12 @@ class JooqTouchstoneRepository(
             record[COVERAGE.TARGET],
             record[COVERAGE.COVERAGE_]
     )
+
+    fun mapDemographicRow(record: Record) = DemographicRow(
+            record[DEMOGRAPHIC_STATISTIC.COUNTRY],
+            "${record[DEMOGRAPHIC_STATISTIC.AGE_FROM]} - ${record[DEMOGRAPHIC_STATISTIC.AGE_TO]}",
+            record[DEMOGRAPHIC_STATISTIC.YEAR],
+            record[GENDER.NAME],
+            record[DEMOGRAPHIC_STATISTIC.VALUE]
+    )
```

Comment:
I haven't traced through every part of these SQL queries in detail. Question: Did you do any performance testing on these in the end? i.e. against the real data? Because I know you spent some time thinking about query optimisation. It seems nice and fast with the test data, but that's very small.